testing/trino-testing/src/main/java/io/trino/testing/sql/TestTable.java

```diff
@@ -31,7 +31,8 @@
         implements AutoCloseable
 {
     private static final SecureRandom random = new SecureRandom();
-    private static final int RANDOM_SUFFIX_LENGTH = 5;
+    // The suffix needs to be long enough to "prevent" collisions in practice. The length of 5 was proven not to be long enough
+    private static final int RANDOM_SUFFIX_LENGTH = 10;
```

Comment:
I recall some issues with too long suffix. Oracle?

Reply:
Yes, there was (your) change from 12 to 5.